### PR TITLE
adding showClearSearchButton property

### PR DESCRIPTION
**Pull request by vlio20**
closes #290

### CODE REVIEW

**Review by macjohnny — APPROVED** [review]
LGTM
thanks!

**Comment by vlio20** [comment]
You are merging it?

**Comment by macjohnny** [comment]
@vlio20 yes, i will release it soon

**Comment by macjohnny** [comment]
Would you mind renaming the property to „hideClearSearchButton“ ?

**Comment by vlio20** [comment]
Done

File: src/app/mat-select-search/mat-select-search.component.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
           diameter="16"></mat-spinner>
 
   <button mat-button
-          *ngIf="value && !searching"
+          *ngIf="hideClearSearchButton && value && !searching"
```
Comment (macjohnny) [inline src/app/mat-select-search/mat-select-search.component.html:37]:
```suggestion
          *ngIf="!hideClearSearchButton && value && !searching"
```

Comment (vlio20) [reply src/app/mat-select-search/mat-select-search.component.html]: great catch!

**Comment by vlio20** [comment]
@macjohnny anything else?

**Pull request merged**